src/powershell-native/nativemsh/pwrshplugin/entrypoints.cpp

```diff
@@ -299,6 +301,7 @@ unsigned int GetCLRVersionForPSVersion(int iPSMajorVersion,
 
 DWORD ReportOperationComplete(WSMAN_PLUGIN_REQUEST *requestDetails, DWORD errorCode)
 {
+    WCHAR szMessage[256] = L"\0";
```

Comment:
After talking to Dan, `szMessage` contains a default error message in the case where the actual error message can't be allocated.  So `szMessage` should be called something else like `szFallbackErrorMessage` making this more clear.  Second, the 256 bytes is arbitrary.  Since we know the max length based on line 315 below, we should only allocate that much.

Reply:
Fixed.
Note that I left the stack buffer at 64.